fwoa-tools/src/binaryConverter.test.ts

```diff
@@ -0,0 +1,299 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as fs from 'fs';
+import AWS from 'aws-sdk';
+import { GetObjectRequest, ListObjectsV2Request, PutObjectRequest } from 'aws-sdk/clients/s3';
+import * as AWSMock from 'aws-sdk-mock';
+import {
+  convertBinaryResource,
+  getBinaryObject,
+  getBinaryResource,
+  parseCmdOptions,
+  uploadBinaryResource
+} from './binaryConverter';
+import { ExportOutput } from './migrationUtils';
+
+AWSMock.setSDKInstance(AWS);
+let fakeFile: ExportOutput = {
+  jobId: 'binaryUnitTests',
+  file_names: { Binary: ['Binary/Binary-0.ndjson'] }
+};
+
+describe('binaryConverter', () => {
+  afterEach(() => {
+    AWSMock.restore();
+  });
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env.BINARY_BUCKET_NAME = 'unit_test_binary_bucket_name';
+    process.env.EXPORT_BUCKET_NAME = 'unit_test_export_bucket_name';
+  });
+
+  describe('parseCmdOptions', () => {
+    beforeEach(() => {
```

Comment:
This beforeEach block is repeated multiple times, why?

Reply:
Consolidated into one block to define all env vars